scripts/deploymentScript.js

```diff
@@ -11,27 +11,28 @@ var webapp = {
 var statusBuild = -1;
 var statusStart = -1;
 
+// Don't need to build web app if NODE_ENV is testing
+if (process.env.NODE_ENV !== 'testing') {
```

Comment:
Also, this `if` statement skipped the `exec('yarn')`. This is essential for the travis to install all the npm dependency. It was lucky that the Travis skipped the if function.